addon-test-support/dom/find-tooltip.js

```diff
@@ -0,0 +1,51 @@
+export function findTooltip(selectorOrElement, { targetSelector, searchMultiple = false } = {}) {
```

Comment:
I think this option should just be shortened to `multiple`

Reply:
Changed it!